js/models/ShippingAddress.js

```diff
@@ -2,7 +2,7 @@ import BaseModel from './BaseModel';
 import app from '../app';
 import is from 'is_js';
 
-export default class extends BaseModel {
+export default class ShippingAddress extends BaseModel {
```

Comment:
It was clear. It was instrumentation. I was seeing if Backbone would read and report the model class name so I could see what NestedClass type name was. Turns out Backbone doesn't keep the names.